src/lib/deploy.js

```diff
@@ -389,7 +394,10 @@ export async function deploy({ alias, yes }) {
   }
 
   let transaction = await step('Build transaction', async () => {
-    let Network = Mina.Network(graphQlUrl);
+    const Network = Mina.Network({
```

Comment:
Just curious. Did you update this api in `o1js`?

Reply:
Yes and released.
https://github.com/o1-labs/o1js/pull/1387
https://github.com/o1-labs/o1js/pull/1389